drracket/scribblings/drracket/extending.scrbl

```diff
@@ -31,8 +31,25 @@ DrRacket supports two forms of extension to the programming
   creating @tech{plugin}s, see @other-manual['(lib
   "scribblings/tools/tools.scrbl")].}
 
+@item{The @tech{Scripts} menu provided by the Quickscript plugin, makes
```

Comment:
Are you sure that `Script` was `deftech`-ed somewhere? The red underline in your screenshot shows that the links is broken.

Reply:
fixed 
![image](https://user-images.githubusercontent.com/108047/83756113-794a2a00-a666-11ea-8265-af256cf06088.png)